userdocs/diagnostics/protocol-type-non-conformance.md

```diff
@@ -52,10 +52,41 @@ In general, any initializers, static members, and associated types required by a
 
 Currently, even if a protocol `P` requires no initializers or static members, the existential type `P` does not conform to `P` (with exceptions below). This restriction allows library authors to add such requirements (initializers or static members) to an existing protocol without breaking their users' source code.
 
+## Exceptions
+
+The Swift protocol `Error` has no required members and, when used as a type, conforms to itself; `@objc` protocols with no static requirements can also be used as types that conform to themselves.
```

Comment:
Perhaps "has no requirements" instead of "has no required members"? Also, it looks like the protocol does [have requirements](https://github.com/apple/swift/blob/923d6ed66b9757d43506e98ce9a1885d53ce688b/stdlib/public/core/ErrorType.swift#L113), but with default implementations and the requirements are underscored. I am not sure what effect that has on self-conformance though.

Reply:
I reworded because requirements include _semantic_ ones, and obviously `Error` has the semantic requirement that conforming types, well, model recoverable errors. We mean to say that it has no _syntactic_ requirements, but you're right that it actually does have a few underscored ones which aren't user-facing, so maybe we could just delete that statement.